sdk/schemaregistry/test-resources.json

```diff
@@ -108,6 +129,159 @@
         "kafkaEnabled": true
       }
     },
+    {
+      "type": "Microsoft.EventHub/namespaces",
```

Comment:
Curious why we need a separate namespace for each schema type - I realize we already have this pattern, but wondering if it would be simpler to just have a single namespace containing all of the schema groups that we need, i.e. 2 for each type.

Reply:
I tried doing this initially but was running into issues when creating the resources, I'll try it again and try to dig into the error more.

Reply:
Well, I tried a lot of different things but keep getting the error:

"The deployment 'xxx' failed with error(s). Showing 1 out of 1 error(s). Status Message: SubCode=40300, Cannot create schema group azsdk_net_test_group_avro because no more than 1 groups is allowed. Currently there are 1 groups for the namespace"

So I think going with the ARM library as mentioned in the other comment, is probably the best option. I don't think using test resources is going to be the best developer experience. The one group per namespace and 25 schemas per group is definitely very limiting.